active_learning/fetch_data.py

```diff
@@ -11,7 +11,18 @@
 # for feature_selection
 from sklearn.feature_selection import SelectKBest, chi2
 
-
+'''
```

Comment:
In python, the convention seems to be to put docstrings inside a function. See [here](https://www.python.org/dev/peps/pep-0257/)